### ISSUE TITLE
Desktop / Browser: Accessibility - TOTP countdown not conveyed to assistive technology users

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#2657]
### Steps To Reproduce

1. Use a screen reader
2. Go to a cipher/item's details view that contains a time-based one-time password (TOTP)
3. Set focus on the "Copy..." button

### Expected Result

Ideally, the countdown for how long the password is valid should be announced dynamically when the copy button has focus.

### Actual Result

The countdown only happens visually and is not conveyed dynamically to assistive technology users. The remaining time is not part of the copy button's announcement, nor is there any other easy way to find out what the remaining time is.

In addition, when navigating using reading keys (cursor keys), the countdown is announced as if it were part of the TOTP code itself (see video below)

### Screenshots or Videos

Video using Chrome/NVDA on Windows, using the browser extension (but the same exact issue happens in the desktop application)

https://user-images.githubusercontent.com/895831/167274116-d46469ab-bf94-4e99-902d-016ce61b00de.mp4

0:00 - 0:11 setting focus on the copy button and letting the countdown run. note that no announcement is made (neither when the button first receives focus, nor when the timer runs out and restarts with a new code)
0:11 - 0:21 using reading keys to go backwards from the copy button. note how the entirety of the item is read, and how the (at that point) current number of remaining seconds is effectively appended to the numerical TOTP code itself that is announced.
0:22 - 0:28 just quickly moving away and back again to the item, to show that the then current number of remaining seconds is again announced at the time the reading focus gets to the item.
0:28 - 0:29 just moving one more time to the copy button

### Additional Context

I'll ponder an appropriate fix for this and submit a PR, but off the top of my head:

- the countdown timer needs to be split out from the item itself (so it's not announced as if it was just part of the code when using reading keys)
- having the countdown as a live region would be too intrusive/distracting, as it would constantly announce the change each second once the item view is displayed
- using an `aria-describedby` on the copy button, referencing the countdown, should work - need to test though if it will dynamically announce a change of the countdown when the button has focus. otherwise, this MAY need extra work to dynamically add/remove `aria-live` on the countdown when the button has focus, and removing it otherwise.

### Operating System

Windows

### Operating System Version

11

### Web Browser

Chrome

### Browser Version

101.0.4951.54 (Official Build) (64-bit)

### Build Version

1.58.0 (extension) / Version 1.33.0 Shell 16.1.0 Renderer 96.0.4664.174 Node 16.9.1 Architecture x64 (desktop)
